mongodb-atlas-sql-interface.md

```diff
@@ -48,17 +48,20 @@ To connect using the Atlas SQL interface:
 
 4. In the MongoDB Atlas SQL window that appears, fill in the following values:
 
-   - The **MongoDB URI**. _Required_.   
+   - The **MongoDB URI**. _Required_ \
```

Comment:
Rather than insert empty spaces at the end to force a newline, it's better to use the backslack. Many markdown formatters will remove trailing spaces, and markdown lint complains about them if left in.